tests/framework/microvm.py

```diff
@@ -82,6 +82,10 @@ def __init__(
         )
         self.jailer_clone_pid = None
 
+        # Copy the /etc/localtime file in the jailer root
+        self.jailer.copy_into_root(
+            "/etc/localtime", create_jail=True)
```

Comment:
What if there's no `/etc/localtime` on the host?

Reply:
this will be handled by installing `tzdata` in the dev container

Reply:
handled